code/code/sys/create_character.cc

```diff
@@ -5,6 +5,7 @@
 //////////////////////////////////////////////////////////////////////////
 
 #include <sstream>
+#include "configuration.h"
```

Comment:
How did it compile before?

Reply:
The MUD_NAME and MUD_NAME_VERS variables were declared in a different header previously, which this file already included. I had to add this once I moved their declarations into `configuration.h`.